flow-libs/parcel-watcher.js.flow

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+// Derived from the README and source of ncp located at
```

Comment:
this isn't relevant for this one 😃

Reply:
Woops, was so happy to get flow passing, I forgot to clean up :)